_sass/elements/_lists.scss

```diff
@@ -18,6 +30,35 @@
 //
 // Styleguide elements.lists
 
+ol,
+ul {
+  padding-left: $base-padding;
```

Comment:
I'm not sure how this slipped by, but not sure it is worth making another PR to dev to fix.